src/llmcompressor/modifiers/modifier.py

```diff
@@ -9,7 +10,7 @@
 __all__ = ["Modifier"]
 
 
-class Modifier(BaseModel, ModifierInterface):
+class Modifier(BaseModel, ModifierInterface, ABC):
```

Comment:
ModifierInterface is ABC, do we need ABC for Modifier?

Reply:
Technically no, but abstract classes and methods are basically type hints anyways, so I think it's nice to keep it explicit